### PR TITLE
chore: remove inline markdown

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,12 +67,12 @@
   </section>
   <section>
     <h2>Introduction</h2>
-    <p>**Performance is a feature.** Modern browsers provide extensive tooling
-    to help developers understand and optimize their applications:
-    network-layer insights about the requests (timing, cache, request and
-    response payload, etc); rendering insights (cost of layout and style
-    recalculations, image decoding, per-layer rendering costs, idle time, etc);
-    CPU and memory profiling; JavaScript debugging, and much more. As a
+    <p><strong>Performance is a feature.</strong> Modern browsers provide
```
Comment (yoavweiss) [inline index.html:70]: Was inline markdown supported and removed?

Comment (marcoscaceres) [reply index.html:70]: “It’s complicated” - but short answer is yes.

Comment (yoavweiss) [reply index.html:70]: That's the kind of breaking change that would make pointing to a fixed version and only upgrade after testing valuable.

**Review by yoavweiss — APPROVED** [review]
LGTM! Thanks for fixing

**Pull request merged**